tests/Doctrine/Tests/ORM/Tools/Console/Command/SchemaTool/CreateCommandTest.php

```diff
@@ -8,10 +8,15 @@
 
 class CreateCommandTest extends AbstractCommandTest
 {
+    /**
+     * @doesNotPerformAssertions
+     */
     public function testItPrintsTheSql(): void
     {
         $tester = $this->getCommandTester(CreateCommand::class);
         $tester->execute(['--dump-sql' => true]);
-        self::assertStringContainsString('CREATE TABLE keyboard', $tester->getDisplay());
+
+        $statements = $tester->getDisplay();
+        self::$sharedConn->executeStatement($statements);
```

Comment:
Now that we've establish that this works, let's get rid of the extra variable, here and below.

```suggestion
        self::$sharedConn->executeStatement($tester->getDisplay());
```